app/assets/stylesheets/govuk_publishing_components/components/_action-link.scss

```diff
@@ -118,19 +118,31 @@
 }
 
 .gem-c-action-link--dark-icon {
+  &:before {
+    width: 30px;
+    // sass-lint:disable no-duplicate-properties
+    background: image-url("govuk_publishing_components/action-link-arrow--dark.png");
+    background: image-url("govuk_publishing_components/action-link-arrow--dark.svg"), linear-gradient(transparent, transparent);
+    // sass-lint:enable no-duplicate-properties
+  }
+}
+
+.gem-c-action-link--small-icon {
+  &:before {
+    width: 35px;
```

Comment:
I'm a bit confused why small is 35px wide and dark is 30px, when the icons for each are the same size. Would it be possible to make them consistent?

Reply:
Yeah the icons are the same size, but the header version requires more visual space between the icon and the text vs the light background/dark text version. 

See mockup (left) vs existing `dark_icon` version (right)

<img width="1552" alt="Screenshot 2020-06-25 at 11 59 05" src="https://user-images.githubusercontent.com/7116819/85707688-577b1a80-b6db-11ea-8b22-9e84b5ef9ece.png">


We could make the spacing consistent with the `dark_icon` one – I just think it might look a bit squished? I can check with Mia if she thinks it looks ok with the smaller spacing though.

Reply:
I was thinking the other way round... the `dark_icon` variant looks a little squished. But yeah, see what Mia thinks. I'm just against having tiny differences that add more code if there's not a good reason to have those differences.

Reply:
Yeah agree with you, I also dislike having these little exceptions for not very good reasons. 😕 
Might be better to increase spacing for the other variant indeed. Will discuss with Mia and get back to you on that.

Reply:
@andysellick I've spoken to Mia and she agrees it would be best to keep things consistent. She suggested that we match the existing dark icon style for now (I've updated the PR to reflect that)